pilot/pkg/features/pilot.go

```diff
@@ -281,14 +281,6 @@ var (
 
 	EnableLazySidecarEvaluation = env.Register("ENABLE_LAZY_SIDECAR_EVALUATION", true,
 		"If enabled, pilot will only compute sidecar resources when actually used").Get()
-
-	// EnableCACRL ToDo (nilekh): remove this feature flag once https://github.com/istio/istio/issues/56529 is closed
-	EnableCACRL = env.Register(
```

Comment:
I'm thinking we should keep the feature flag and just default it to true in case it breaks in some way. That way, users can turn the feature off

Reply:
I have re-added the feature flag with the default set to true.

Reply:
Yeah we can remove it in a release or two once we've ensured its stability

Reply:
@keithmattix if a feature isn't stable can it be enabled by default?